src/lib/viewers/BaseViewer.js

```diff
@@ -545,6 +545,9 @@ class BaseViewer extends EventEmitter {
      */
     handleFullscreenEnter() {
         this.resize();
+        if (this.annotator) {
+            this.annotator.emit(ANNOTATOR_EVENT.toggleVisibility);
```

Comment:
This may be a silly question, but if the visibility is toggled off for some other reason, this will turn them back on. Should we consider passing in an explicit boolean value, instead?

Reply:
> This may be a silly question, but if the visibility is toggled off for some other reason, this will turn them back on. Should we consider passing in an explicit boolean value, instead?

Yeah, toggleMode also has this problem. Should I change them together?

Reply:
I think the toggleMode method is already present in the 2.x and 3.x version of annotations, which is why it's still a direct method invocation.